taipy/core/_orchestrator/_orchestrator_factory.py

```diff
@@ -61,9 +61,9 @@ def _build_dispatcher(cls, force_restart=False) -> Optional[_JobDispatcher]:
         return cls._dispatcher
 
     @classmethod
-    def _remove_dispatcher(cls) -> Optional[_JobDispatcher]:
+    def _remove_dispatcher(cls, wait: bool = True, timeout: Optional[float] = None) -> Optional[_JobDispatcher]:
```

Comment:
This function always returns None.
```suggestion
    def _remove_dispatcher(cls, wait: bool = True, timeout: Optional[float] = None) -> None:
```

Reply:
Good catch